controllers/articles.controllers.js

```diff
@@ -0,0 +1,13 @@
+const fetchArticleById = require('../models/articles.models')
+
+getArticleById = (req, res) => {
+    const { article_id } = req.params
+    fetchArticleById(article_id).then((article) => {
+        res.status(200).send({article: article})
```

Comment:
Nothing wrong with this but if the key is the same as the variable you can use the shorthand of {article}